### PR TITLE
Fix 178, should not fail when there are no 'example' for a component

### PR DESCRIPTION
**Pull request by gervaisb**
This fix the issue #178 by excluding poperties with a defined `allOf` from the example validation.

### CODE REVIEW

**Comment by JFCote** [comment]
@gervaisb Thanks for your contribution. I need some time to check this and right now I'm in the middle of a big project. Will check this in 2 weeks.

**Comment by JFCote** [comment]
@gervaisb  I did not forget about you. Just been very busy ! I'm also having problem with the sonarqube validation that is done on each commit. Checking this with the people of openapitools who manage the organization. Will try to get back to you as soon as possible. Sorry about the delay!

File: lib/src/main/java/org/openapitools/openapistylevalidator/OpenApiSpecStyleValidator.java [hunk 1]
```diff
@@ -131,7 +131,8 @@ private void validateModelProperties(String modelName, Schema model) {
             model.getProperties().forEach((propertyName, property) -> {
                 if (parameters.isValidateModelPropertiesExample() && property.getExample() == null
                         && ((property.getItems() == null && property.getRef() == null)
-                            || (property.getItems() != null && property.getItems().getRef() == null))) {
+                            || (property.getItems() != null && property.getItems().getRef() == null)) &&
+                            property.getAllOf()==null) {
```
Comment (JFCote) [inline lib/src/main/java/org/openapitools/openapistylevalidator/OpenApiSpecStyleValidator.java:135]: Please do a little reformat here with space before and after the `==`.
Like this:
```suggestion
                            property.getAllOf() == null) {
```

**Review by JFCote — CHANGES_REQUESTED** [review]

**Review by JFCote — APPROVED** [review]

**Comment by gervaisb** [comment]
Hey, thank you for your review and merge.

Any idea when this will be released ?

**Comment by JFCote** [comment]
@gervaisb I will check with @jmini . He's the one with the key ready for the release.

**Comment by JFCote** [comment]
@gervaisb Should be done within the end of the week for version 1.7

**Comment by JFCote** [comment]
@gervaisb It's been released!!

**Pull request merged**